cli-plugins/apps/troubleshooting/troubleshooting.hbs.md

```diff
@@ -0,0 +1,6 @@
+# Troubleshooting in Apps CLI Plug-in
+
+The Troubleshooting section contains the following topics:
+
+- [Troubleshooting workloads](troubleshooting-workload.hbs.md)
+- [Troubleshooting LSP integration](troubleshooting-workload.hbs.md#local-source-proxy-integration)
```

Comment:
Rename this to [Troubleshooting Local source Proxy integration]

Reply:
renamed